app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardResultsStep/helpers.js

```diff
@@ -0,0 +1,20 @@
+import { SOURCE_HREF_SLUGS, DESTINATION_HREF_SLUGS } from './MappingWizardResultsStepConstants';
+
+const normalizeTransformationItem = (transformationItem, targetProvider) => ({
+  source: SOURCE_HREF_SLUGS[transformationItem.source_type] + transformationItem.source_id,
+  destination:
+    DESTINATION_HREF_SLUGS[targetProvider][transformationItem.destination_type] + transformationItem.destination_id
```

Comment:
Can we use interpolation here like we normally do?

```js
`${DESTINATION_HREF_SLUGS[targetProvider][transformationItem.destination_type]}${transformationItem.destination_id}`
```

Reply:
Hahaha, yes :) I did it that way first, but it felt funny b/c I wasn't adding any strings between or around the variables.  I'll update

Reply:
For some reason, everyone seems to prefer interpolation over string math :)

I don't have a preference as such, but would just like to see things uniform.